notebooks/NLP_DL.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
For all notebooks, I would use "Explain your model" instead of "Make intelligibility"